.ci/pitest-survival-check-xml.groovy

```diff
@@ -261,10 +261,12 @@ private static int compareMutations(Set<Mutation> survivingMutations,
     final int exitCode
     if (survivingMutations == suppressedMutations) {
```

Comment:
Can these 2 lists be not equal but still be empty?

If so, then the `else` portion still doesn't display something in it's area.

Reply:
If they are empty, they will be equal, both are guaranteed to be `TreeSet`s.

Reply:
https://stackoverflow.com/questions/9682206/groovy-different-results-on-using-equals-and-on-a-gstringimpl
> In groovy a == b checks first for a compareTo method and uses a.compareTo(b) == 0 if a compareTo method exists. Otherwise it will use equals.

Reply:
For a `Set<T>` it invoked `compareTo` on the object of type `T` and in our case, we implement comparable so `AbstractSet`s `equals()` won't be called.